content/SCALE/SCALETutorials/Apps/MinIOClustersSCALE/MinIOClustering.md

```diff
@@ -13,7 +13,7 @@ tags:
 
 On TrueNAS SCALE 20.12-ALPHA and later, users can create a MinIO S3 distributed instance to scale out and handle individual node failures. A node refers to a single TrueNAS storage system in a cluster.
 
```

Comment:
I think we need to define node somewhere in this article. While the reader might know what we mean, we should be clear about what we mean when we say node.
We should include this definition in any article that uses the term.